build.gradle

```diff
@@ -12,6 +12,9 @@ repositories {
 dependencies {
     testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.10.0'
     testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.10.0'
+
+    testImplementation 'org.mockito:mockito-core:5.3.1'
+    testImplementation 'org.mockito:mockito-junit-jupiter:5.3.1'
```

Comment:
I saw from this link: https://github.com/nus-cs2113-AY2425S1/forum/issues/36
Mockito library is approved for use in tP!

Reply:
yes that is great !